scripts/proxySetup.sh

```diff
@@ -27,9 +27,9 @@ InstallPrereqs() {
     maxRetries=3
     retryCount=0
     installSucceeded=1
-    sudo apt-get -y update >> install.log 2>&1
-
-    apt update >> install.log 2>&1
+    
+    apt-get -y update >> install.log 2>&1
+    apt-get -y install apache2-utils >> install.log 2>&1
```

Comment:
This should be in the retry loop with docker since the reason for the retry loop was pre-req's not installing due to a race condition.

Reply:
Fixed